client/coral-ui/components/Checkbox.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import styles from './Checkbox.css';
 
-export default ({name, cStyle = 'base', onChange, label, className, info, checked = 'false'}) => (
+export default ({name, cStyle = 'base', onChange, label, className, info, ...props}) => (
```

Comment:
It's not the same that we discussed with Riley and you the other day.

The reason behind using `...props` here it's because if I want to use `defaultValue` or `value` (meaning that I want an Uncontrolled or a Controlled Component) I need to use one of them. And that is the prop that will finally get passed to the input. 

This is a UI component. I should be able to pass as many custom props (attributes) as needed. It's not the same as the Components that we have for Stream and such. Those are not UI Components, meaning no custom props passed.